confidant/routes/credentials.py

```diff
@@ -180,7 +180,11 @@ def get_credential(id):
 
     permissions = {
         'metadata': True,
-        'get': False,
```

Comment:
Question: what does True and False mean here?

Reply:
True means that they're able to view certain UI components that allow the user to decrypt

eg:
https://github.com/lyft/confidant/blob/eeff4705cc29b6c8f873860378e33956af1fad24/confidant/public/modules/resources/views/credential-details.html#L54

Note this is only UI control.  The backend server ultimately allows / denies the decryption.